hcloud/network.go

```diff
@@ -29,8 +29,15 @@ type NetworkSubnetType string
 
 // List of available network subnet types.
 const (
-	NetworkSubnetTypeCloud   NetworkSubnetType = "cloud"
-	NetworkSubnetTypeServer  NetworkSubnetType = "server"
+	// Used to connect cloud servers.
```

Comment:
This subnet type also allows Cloud Load Balancers

```suggestion
	// Used to connect cloud servers and load balancers.
```

Reply:
I'll update this in the spec first.